shared/bazel/rules/cc_flags.bzl

```diff
@@ -0,0 +1,49 @@
+def get_wpigui_linker_flags(console):
```

Comment:
Can these go on the `linkopts` in `//wpigui` (or even better on the imgui library or whatever else directly depend on these) so they don't have to be specified everywhere?

Reply:
I popped it on `wpigui` and it compiles. I don't have a way to test on mac, but it didn't link without those flags, and does if I put it there.